UPGRADE-4.0.md

```diff
@@ -184,6 +184,31 @@ Form
    }
    ```
 
+ * Using the "choices" option in ``CountryType``, ``CurrencyType``, ``LanguageType``,
+  ``LocaleType``, and ``TimezoneType`` is now ignored. Use the "choice_loader" option
+   instead or explicitly set it to ``null``.
```

Comment:
@xabbuh supporting a single way to achieve it would actually involve **more** work to forbid one of them. Both ways are just features of the ChoiceType: if ``choice_loader`` is not null, it is used. Otherwise ``choices`` is used.

The CountryType & co currently have to do extra work to ensure that this priority order is reversed, due to BC.